portfolio/src/main/webapp/paragraph.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Adds a headig with a paragraph to each picture.
+ */
+let files = getImages("/images");
+let descriptions = ["This picture was taken accidentally by my friend two years ago. However it becomes one of my favorite photo since at that time I was still quite slim.",
```

Comment:
Actually the images are returned from one place, and the matching descriptions are created here. It's hard to maintain the consistency this way.
Maybe the other method can return a map? e.g {imageName: description}?
and you can have this method creating divs that contain images and descriptions?

Reply:
Fixed